player/lua/stats.lua

```diff
@@ -1803,15 +1757,8 @@ if o.bindlist ~= "no" then
     mp.set_property_bool("msg-time", false)
     -- wait for all other scripts to finish init
     mp.add_timeout(0, function()
-        if o.bindlist:sub(1, 1) == "-" then
-            o.bindlist = o.bindlist:sub(2)
-            o.no_ass_b0 = ""
-            o.no_ass_b1 = ""
-        end
-        local width = max(40, math.floor(tonumber(o.bindlist) or 79))
         o.ass_formatting = false
         o.no_ass_indent = " "
-        o.term_size = { w = width , h = 0}
```

Comment:
I think it should be kept as `o.term_size = { h = 0 }` we don't want any scrolling to happen.

Reply:
Where is this used? No scrolling happens without it.